tests/Get-sbRDSession.Tests.ps1

```diff
@@ -27,5 +27,17 @@ InModuleScope $ModuleName {
         It "only accepts active, idle, disconnected, connected or any" {
             { Get-sbRDSession -SessionState Dancing } | Should Throw
```

Comment:
You can validate the scenarios that should pass too, like:
```
{ Get-sbRDSession -SessionState Active} | Should Not Throw
```
I've not tested the above, but you get the gist.

Reply:
Noted and added.